projects/experimental/components/badge/badge.template.html

```diff
@@ -0,0 +1,20 @@
+<div class="t-icon">
+    <ng-content></ng-content>
+</div>
+<div class="t-image">
+    <ng-content select="img"></ng-content>
+</div>
+<tui-fade
+    *ngIf="!isEmpty"
+    class="t-fade"
+    [size]="textOverflow(text) ? '1.5rem' : '0'"
```

Comment:
We do we need this? It shouldn't have any fade effect when it's not overflown out of the box.